helpdesk_mgmt_ticket_closed_date/models/helpdesk_ticket.py

```diff
@@ -0,0 +1,31 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import models, api, _, fields
+from odoo.exceptions import ValidationError
+
+
+class HelpdeskTicket(models.Model):
+    _inherit = "helpdesk.ticket"
+
+    # default=True bug
+    closed_date_permits = fields.Boolean(
+        compute="_compute_closed_date_permits"
+    )
+
+    def _compute_closed_date_permits(self):
+        if (
+            self.stage_id.closed and
+            not self.env.user.has_group('helpdesk_mgmt.group_helpdesk_manager')
+        ):
+            self.closed_date_permits = False
+        else:
+            self.closed_date_permits = True
```

Comment:
Code optimization, singleton error prevention, and an slightly different condition (prevent `closed_date` edition whether ticket is not closed, even if user is manager):
```suggestion
        is_manager = self.env.user.has_group('helpdesk_mgmt.group_helpdesk_manager')
        for record in self:
            record.closed_date_permits = record.stage_id.closed and is_manager
```
The proposed condition has one problem: attepting to change `closed_date` when the ticket is not closed (but it's filled due to ticket was closed before). In that case, could you check if standard unsets closed_date, or it remains filled?